### PR TITLE
Updated rubygem-smart_proxy_pulp to version 1.2.0

### PR DESCRIPTION
**Pull request by witlessbird**

### CODE REVIEW

File: rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 Summary: Basic Pulp support for Foreman Smart-Proxy
 Name: rubygem-%{gem_name}
-Version: 1.0.1
+Version: 1.2.0
 Release: 2%{?dist}
```
Comment (domcleal) [inline rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec]: reset this to 1

File: rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec [hunk 2]
```diff
@@ -79,6 +79,9 @@ cp -pa .%{gem_instdir}/settings.d/pulpnode.yml.example %{buildroot}%{foreman_pro
 
 
 %changelog
+* Wed Jan 27 2016 Dmitri Dolguikh <dmitri@appliedlogic.ca> 1.2.0
```
Comment (domcleal) [inline rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec]: add a `-1` to the version string, e.g. `1.2.0-1` to match the package's EVR

**Comment by witlessbird** [comment]
@domcleal: fixed!

**Comment by domcleal** [comment]
Merged, thanks @witlessbird.  Package is building to the nightly repos, please let me know if you'd like it available to older Foreman releases (there is a repo per major release).

**Pull request merged**